public/app/features/alerting/unified/components/rules/AlertStateTag.tsx

```diff
@@ -12,9 +13,18 @@ interface Props {
   muted?: boolean;
 }
 
-export const AlertStateTag = React.memo(({ state, isPaused = false, size = 'md', muted = false }: Props) => (
-  <StateTag state={alertStateToState(state)} size={size} muted={muted}>
-    {alertStateToReadable(state)} {isPaused ? ' (Paused)' : ''}
-  </StateTag>
-));
+export const AlertStateTag = React.memo(({ state, isPaused = false, size = 'md', muted = false }: Props) => {
+  if (isPaused) {
+    return (
+      <Tooltip content={'Alert evaluation is currently paused'} placement="top">
+        <Tag icon="pause" name="Paused" colorIndex={1} />
+      </Tooltip>
+    );
+  }
+  return (
+    <StateTag state={alertStateToState(state)} size={size} muted={muted}>
```

Comment:
`StateTag` doesn't use the `@grafana/ui` Tag, and I didn't want to start touching that _as well_ so I just decided to render a different tag + tooltip altogether when necessary

Reply:
I think we can re-use `StateTag` here like so

```
<Tooltip content={'Alert evaluation is currently paused'} placement="top">
  <StateTag state="warning" size={size} muted={muted}>
    <Stack direction="row" gap={0.25} alignItems="center">
      <Icon name="pause" size="xs" /> Paused
    </Stack>
  </StateTag>
</Tooltip>
```

<img width="366" alt="image" src="https://github.com/grafana/grafana/assets/868844/3b1cc251-398a-447f-bcdc-c90016616c63">

Reply:
Sure - have done this now. I had to wrap the existing `StateTag` in a `forwardRef` as otherwise the tooltip doesn't work. 

I could have (instead?) changed the `StateTag` to use `Tag` internally, but the styling isn't exactly the same - the contrasting text colour isn't the same etc., and `Tag` doesn't have different sizes (but that highlights that our tags are slightly different from the main grafana/ui ones 🤷 )